modules/accredited_representative_portal/app/sidekiq/accredited_representative_portal/allow_list_sync_job.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class AllowListSyncJob
+    include Sidekiq::Job
+    sidekiq_options retry: false
+
+    # To not overload the DB.
+    MAX_RECORD_COUNT = 500
+
+    Error = Class.new(RuntimeError)
+
+    class RecordCountError < Error
+      def initialize(record_count)
+        super("record_count: #{record_count}")
+      end
+    end
+
+    def perform
+      SemanticLogger.tagged do
+        csv = extract
+        csv.size.between?(1, MAX_RECORD_COUNT) or
+          raise RecordCountError, csv.size
+
+        attributes = transform!(csv)
+        result = load(attributes)
+
+        logger.info(result)
+      rescue => e
+        logger.error(e)
```

Comment:
What do you plan to do with this error? Are you going to be looking out for it in Datadog? Would it help to prefix it with something like, `"AllowListSyncJob error"`? I'm just thinking it might be easier to find in Datadog.

Reply:
@rmtolmach We checked log output locally and saw that the `SemanticLogger.tagged` wrapper prepends Sidekiq job name. Does that sound familiar? We could look into this question further.

Reply:
@rmtolmach [this middleware it looks like](https://github.com/department-of-veterans-affairs/vets-api/blob/271d7684a7367d11d07ba1dc33e2e846c7f465ac/lib/sidekiq/semantic_logging.rb)